docs/app/pages/components/components-sections/date-time-picker/date-range-picker/date-range-picker.component.html

```diff
@@ -152,6 +152,9 @@
     <tr uxd-api-property name="selectEndTitle" type="string" defaultValue="Select End Date">
         Defines the title to display above the end picker.
     </tr>
+    <tr uxd-api-property name="durationTitle" type="(days: number) => string" defaultValue="X days">
+        Defines the title to indicate the duration of the selected range.
```

Comment:
I would suggest changing this to "Defines a function which returns a string indicating the duration of the selected range."

Reply:
Thanks for the feedback, updated the description